docs/3.0/authorization.md

```diff
@@ -6,19 +6,24 @@ license: pro
 
 When you share access to Avo with your clients or large teams, you may want to restrict access to a resource or a subset of resources. One example may be that only admin-level users may delete or update records.
 
-By default, Avo leverages Pundit under the hood to manage the authorization.
+By default, Avo's authorization mechanism is disabled. However, Avo provides a Pundit client out of the box for authorization management, which can be easily enabled.
 
 :::info Pundit alternative
-Pundit is just the default choice. You may plug in your own client using the instructions [here](#custom-authorization-clients).
+Pundit is just one choice among many. You may plug in your own client using the instructions [here](#custom-authorization-clients).
```

Comment:
Pundit is just the default client. You may plug in your own client using the instructions [here](#custom-authorization-clients).
You can use [this](https://github.com/avo-hq/avo/issues/1922) `action_policy` client as well.